autosklearn/pipeline/components/feature_preprocessing/select_rates_classification.py

```diff
@@ -99,15 +105,16 @@ def get_hyperparameter_search_space(dataset_properties=None):
         alpha = UniformFloatHyperparameter(
             name="alpha", lower=0.01, upper=0.5, default_value=0.1)
 
+        if dataset_properties is not None and 'sparse' in dataset_properties \
```

Comment:
You could simplify this to `if dataset_properties is not None and dataset_properties.get('sparse'):`

Reply:
thanks, I added this change.